src/js/components/awardv2/visualizations/amounts/AwardAmounts.jsx

```diff
@@ -45,7 +46,14 @@ export default class AwardAmounts extends React.Component {
         ];
 
         const awards = this.props.overview;
-        const content = this.state.active === 'awards' ? (<AwardAmountsContainer awardId={this.props.awardId} />) : (<IDVAmounts awards={awards} />);
+        const content = this.state.active === 'awards' ? (
+            <AwardAmountsContainer
+                jumpToSection={this.props.jumpToSection}
+                awardId={this.props.awardId} />
+        ) : (
+            <IDVAmounts
+                awards={awards} />
+        );
         return (
```

Comment:
Very clean ternary. Love it!! 💯 

Also wondering in principle, if we're rendering a container, should we be passing props from the parent component if the value of the prop is derived directly from redux?

I guess this would be a question around principle which may not really matter to us, but I'm curious if we'd like to develop a consensus for this scenario as well?

My gut is container components should refer to redux directly via `connect` if possible.

Reply:
Yep, changed `AwardAmountsContainer` to get the award id from Redux.